server/client_server.go

```diff
@@ -22,10 +30,120 @@ func newServerClient(jiraClient *jira.Client) Client {
 	}
 }
 
+type IssueInfo struct {
+	Values []*jira.MetaIssueType `json:"values,omitempty"`
+}
+
+type FieldInfo struct {
+	Values []interface{} `json:"values,omitempty"`
```

Comment:
To make other code suggestions below work

```suggestion
	Values []map[string]interface{} `json:"values"`
```